src/modules/wallet/dto/batch-transfer-wallet.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { IsString, IsNumber, IsArray, ValidateNested } from 'class-validator';
+import { Type } from 'class-transformer';
+
+class CsvJsonDto {
+  @IsString()
+  wallet_name: string;
+
+  @IsNumber()
+  token_transfer_amount_overwrite?: number;
+}
+
+export class BatchTransferWalletDto {
```

Comment:
Also, can you crosscheck and confirm the validations? If any field is required, use IsNotEmpty; if not, use IsOptional decorators. Also, verify if there are any other types of validations applied. https://github.com/Greenstand/treetracker-wallet-api/blob/bf235b0f8e7d9709708681eeadea53880b15f293/server/handlers/walletHandler/schemas.js#L63-L100

For instance, wallet_name should be unique in the csvJson array. And there is no corresponding csvJsonArray validation for BatchCreateWallet